docs/pages/management/diagnostics/metrics.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Metrics
+description: How to enable and consume metrics
+---
+
+## Prerequisites
```

Comment:
```suggestion
## Enabling metrics
```
Since this isn't a step-by-step guide, I think it would be clearer to name this section after its contents.